### PR TITLE
[Fix] Added key accessibility for most LFD features

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6423

## Description
Added key accessibility for most LFD features

## Screenshots/screencasts
https://share.getcloudapp.com/KourrkeE

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: js/layout-javascript/agenda-code.js [hunk 1]
```diff
@@ -669,7 +714,11 @@ DynamicList.prototype.attachObservers = function() {
         });
       }
     })
-    .on('click', '.dynamic-list-delete-item', function() {
+    .on('click keydown', '.dynamic-list-delete-item', function(event) {
+      if (event.type === 'keydown' && event.keyCode !== 13 && event.keyCode !== 32) {
```
Comment (tonytlwu) [inline js/layout-javascript/agenda-code.js:718]: We repeat this so much. Can we simplify this throughout?

```suggestion
      if (!_this.Utils.Event.isExecute(event)) {
```

**Comment by YaroslavOvdii** [comment]
@tonytlwu added suggestions.

File: js/layout-javascript/agenda-code.js [hunk 2]
```diff
@@ -457,7 +486,11 @@ DynamicList.prototype.attachObservers = function() {
         allowClick = true;
       }, 100);
     })
-    .on('click', '.agenda-list-item', function(event) {
+    .on('click keydown', '.agenda-list-item', function(event) {
+      if (event.type === 'keydown' && event.keyCode !== 32 && event.keyCode !== 13) {
```
Comment (tonytlwu) [inline js/layout-javascript/agenda-code.js:490]: There is more in this PR that still needs replacing.

```suggestion
      if (!_this.Utils.Event.isExecute(event)) {
```

**Comment by YaroslavOvdii** [comment]
Created a new [PR ](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/416) to solve conflicts.

**Pull request closed without merging**